sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -374,9 +384,11 @@ private boolean matchStatus(PollResponse<T> currentPollResponse, OperationStatus
      * We will use  {@link PollResponse#getRetryAfter} if it is greater than zero otherwise use poll interval.
      */
     private Duration getCurrentDelay() {
-        return (this.pollResponse != null
-            && this.pollResponse.getRetryAfter() != null
-            && this.pollResponse.getRetryAfter().toNanos() > 0) ? this.pollResponse.getRetryAfter() : this.pollInterval;
+        final PollResponse<T> current = pollResponse.get();
```

Comment:
`this.`  ?

Reply:
Not necessary. There isn't a conflict in member names versus variable names in this scope.